INSTRUCTION.md

```diff
@@ -0,0 +1,23 @@
+# ToDo App in Docker
+
+## Docker Hub Repository
+[Docker Hub Repository Link](https://hub.docker.com/repository/docker/beliar25/todoapp2.0/general)
+[Mysql](https://hub.docker.com/repository/docker/beliar25/mysql-test-db/general)
+
+## Build and Run Instructions
+
+### Build the Docker Image
+To build the Docker image, run the following command:
+```bash
+# Run App container
+docker run -p 8080:8080 --name app app:1.0
```

Comment:
The command to run the app container is placed before the command to build the app image. Ensure that the build command (line 16) precedes the run command to avoid errors if the image is not already built.